Pet-Clinic-Frontend/src/pages/Landing.tsx

```diff
@@ -1,19 +1,16 @@
 import './Landing.css';
 import clinicImage from "../images/petclinic.png";
 
-function PageView() {
+const Landing = () => {
+
   return (
-    <>
-   
-    <img src={clinicImage} alt="Clinic" className="clinic-image" /> 
+    <div>
```

Comment:
The react fragment should be sufficient here.

Reply:
Ok. It is only about code cleanliness?  <Div> is redundant?

Reply:
well it is rather about "semantic html", we rather should not inject more divs then necessary